sources/PTYSession.m

```diff
@@ -3888,24 +3888,69 @@ - (void)appendBrokenPipeMessage:(NSString *)unpaddedMessage {
         }
         screen_char_t savedFgColor = [terminal foregroundColorCode];
         screen_char_t savedBgColor = [terminal backgroundColorCode];
-        // This color matches the color used in BrokenPipeDivider.png.
-        [terminal setForeground24BitColor:[NSColor colorWithCalibratedRed:70.0/255.0
-                                                                    green:83.0/255.0
-                                                                     blue:246.0/255.0
-                                                                    alpha:1]];
+        NSColor *messageColor = [iTermProfilePreferences colorForKey:KEY_SESSION_END_MESSAGE_COLOR inProfile:self.profile];
+        if (!messageColor) {
+            messageColor = [NSColor colorWithCalibratedRed:70.0/255.0
+                                                     green:83.0/255.0
+                                                      blue:246.0/255.0
+                                                     alpha:1];
+        }
+        [terminal setForeground24BitColor:messageColor];
         [terminal setBackgroundColor:ALTSEM_DEFAULT
                   alternateSemantics:YES];
         [terminal updateDefaultChar];
         mutableState.currentGrid.defaultChar = terminal.defaultChar;
+        
+        NSString *dividerStyle = [iTermAdvancedSettingsModel sessionEndMessageDividerStyle];
+        BOOL showDividers = ![dividerStyle isEqualToString:@"none"];
         int width = (mutableState.width - message.length) / 2;
-        if (width > 0) {
-            [mutableState appendNativeImageAtCursorWithName:@"BrokenPipeDivider"
-                                                      width:width];
+        
+        if (showDividers && width > 0) {
+            // Determine divider character based on style
+            NSString *dividerChar = @"━"; // default double line
```

Comment:
The setting could simply be the character rather than the name that describes it. Then you can use anything you can think of.